src/amber/cli/templates/app/shard.yml.ecr

```diff
@@ -22,29 +22,35 @@ dependencies:
     #version: <%= Amber::VERSION %>
 
   granite:
-    github: amberframework/granite
-    version: ~> 0.17.0
+    github: bcardiff/granite
+    branch: crystal/0.31.0
+    # TODO wait for https://github.com/amberframework/granite/pull/363
```

Comment:
@bcardiff thanks!  I will do a deployment of Granite.

Reply:
Cool FYI. Crystal 0.31.0 is on its way. But everything seems ok from my end.

Reply:
Granite 0.17.3 released.